services/authenticator.js

```diff
@@ -5,13 +5,16 @@ const chalk = require('chalk');
 const inquirer = require('inquirer');
 const api = require('./api');
 const { parseJwt } = require('../utils/authenticator-helper');
+const { terminate } = require('../utils/terminator');
 const logger = require('./logger');
 
 // NOTICE: The forest password should be composed of digit, at least on capital letter
 //         and one lower case letter. It also accepts special characters except whitespaces.
 const FORMAT_PASSWORD = /^(?=\S*?[A-Z])(?=\S*?[a-z])(?=\S*?[0-9])\S{8,}$/;
 
 function Authenticator() {
+  const unexpectedError = 'An unexpected error occurred. Please reach out for help in our Slack community or create a Github issue with following error:';
```

Comment:
[Wording] This message is duplicated (some of them contain typos) in :
- lumber-generate.js
- services/authenticator.js
- services/project-creator.js

We need to factorize this in a messages file.